task_02/index.html

```diff
@@ -1,11 +1,195 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-</head>
-<body>
-    Task 2
-</body>
-</html>
\ No newline at end of file
+    <link rel="stylesheet" href="style.css" />
```

Comment:
Same as for meta tags

https://www.w3schools.com/tags/tag_link.asp

Reply:
Will be fixed.